cachemgr/src/test/java/com/bazaarvoice/emodb/cachemgr/invalidate/LocalDataCenterEndPointProviderTest.java

```diff
@@ -41,6 +41,7 @@
 
     @BeforeMethod
     private void setup() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00112)